contracts/contracts/strategies/aerodrome/AerodromeAMOStrategy.sol

```diff
@@ -351,7 +351,7 @@ contract AerodromeAMOStrategy is InitializableAbstractStrategy {
      */
     function depositAll() external override onlyVault nonReentrant {
         uint256 _wethBalance = IERC20(WETH).balanceOf(address(this));
-        if (_wethBalance > 0) {
+        if (_wethBalance > 1e12) {
```

Comment:
🟣 There's a slightly ugly bit with skipping deposits on small amounts. This means that we don't emit external events on some funds that are received from the vault. We don't have any solvency or balance tracking issues here, just emitted logs not lining up by dust amounts.

I guess this okay because any alternative might double count.